jobs/delorean/jenkinsfiles/discovery/rc/github/Jenkinsfile

```diff
@@ -91,7 +51,7 @@ String getAmqOnlinePrerelease(projectOrg, projectRepo, githubCredentialsID) {
 
     dir('amq_online') {
         gitCheckoutRepo(amqOnlineGitUrl, amqOnlineMasterBranch, githubCredentialsID, '.')
-        amqOnlineLatestRC = sh(returnStdout: true, script: "git for-each-ref refs/remotes/ --format='%(refname:short)' | ${rcFilter} | head -n 1").trim()
+        amqOnlineLatestRC = sh(returnStdout: true, script: "git for-each-ref refs/remotes/origin --format='%(refname:short)' | ${rcFilter} | sort -r | head -n 1").trim()
```

Comment:
This was incorrectly returning the amq-online-10 branch. This will just sort the refs and return the one with the largest value. This will fail at a certain point, but does at least work up until a minor value of 10+